### ISSUE TITLE
Add a key argument for workflow.init_glacier_directories() function to select centerline type

### ISSUE THREAD

**Issue by Keeptg** [OGGM/oggm#1356]
The type of `elev_bands` centerline has been support, but the prepare data can only be used by typing `prepro_base_url`. I'm wondering if it's neccessary of adding a kwargs (`fl_type` for example) in the `workflow.init_glacier_directories()` function, so the `elev_bands` centerline can be easily used as the original `centerlines` centerline.

**Comment by fmaussion** [comment]
I'm not exactly sure to understand what you mean, but if I understand well enough, you are suggesting to have a kwarg similar to e.g. `prepro_border` to select from an url.

While this is a good idea, this would imply a very fixed and structured url structure, which would be the same for all types of pre-processed dirs. This might work, but it will imply quite a change and some work I think?

**Comment by Keeptg** [comment]
> you are suggesting to have a kwarg similar to e.g. `prepro_border` to select from an url.

Yes, that's exactlly what I want to say! I'm planing to use `format()` function pass the new kwargs to url directly. It  seems don't need too much work (but I'm not sure). I'll have a try, and submit a PR if I success.

**Comment by fmaussion** [comment]
What I mean is that I think this requires formalizing the gdir structure online in a different way. Take this example for example:

https://cluster.klima.uni-bremen.de/~oggm/gdirs/oggm_v1.4/L3-L5_files/CRU/elev_bands/qc3/pcp2.5/ 

There are three different set-ups in this folder, all with elev_bands. So a user still needs to tell OGGM the full url. What is the added value of such a keyword then?
